src/infrastructure/jira/jira-client/jira-client.ts

```diff
@@ -160,7 +160,7 @@ class JiraClient {
 				connectInstallation.baseUrl,
 			);
 
-			await axios.put(url.toString(), value, {
+			await axios.put(url.toString(), JSON.stringify(value), {
```

Comment:
@akostevich-atlassian I tried this implementation originally but noticed it encodes the string as "\"CONFIGURED\"". Can you confirm the value that get set in app properties as expected? The spec if still a bit vague on this so you may need to talk to the incredibles team.

Reply:
Thank you, Sam, for clarifying this! I've updated implementation according to the contact proposed by the Incredibles team.

Reply:
Did we test this with the other issue properties we write to? Specifically the `attached-design-url-v2` property as I know there's some funky double `JSON.stringify` business going on with that and just want to make sure we haven't broken anything.

Reply:
Thanks for bringing this to my attention to this, Jon! Yes, the PR does not change the behavior of setting `attached-design-url-v2` -- as before, funky double `JSON.stringify` business is still going on :)

We've researched this with Costa but just haven't had time and a good change to apply this refactoring before.